package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "setup-java",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
I think we need to update the version according to the next release version.

Reply:
yes, we are planning to release after this PR merge

Reply:
Could you please change version 2.0.1 to the planing one ?

Reply:
@dmitry-shibanov , please share planned version